### ISSUE TITLE
[FireMonkey] 2.35 response.json doesn't exist

### ISSUE THREAD

**Issue by linsui** [erosman/support#386]
I'm using GM.fetch() to fetch a json file.

```
GM.fetch(listURL, {responseType: "json"}).then(data => data.json)
```

This worked in 2.34 but is broken in 2.35. I can get the file content with `data.text` but `data.json` is undefined. Could you please take a look? Thanks!

**Comment by erosman** [comment]
Oops ... an oversight when adding new features in v2.35.. fixed in v2.36
v2.36 uploaded

**Comment by linsui** [comment]
Thanks!
